### ISSUE TITLE
Bug: [file-extension-in-import] False positive with zx

### ISSUE THREAD

**Issue by Zamiell** [eslint-community/eslint-plugin-n#131]
### Environment

Consider `zx`, which is [a Node.js library by Google to write scripts](https://github.com/google/zx).

You use it by just adding this to the top of your JavaScript/TypeScript file:

```ts
import "zx/globals";
```

However, this line triggers the `file-extension-in-import` rule. I think that this rule is intended to apply to files in your local project, not to imports from 3rd-party packages. So, I think this is a false positive.

In the meantime before this bug is fixed, does anyone know if there is some way to use the rule options to work around this?

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by scagood** [comment]
I think this is a duplicate of #21.
I remember seeing #32 to fix that.

**Comment by Zamiell** [comment]
@scagood Thank you! That's fantastic, it looks like #32 takes care of it.
Looks like the PR has been sitting since August though. @aladdin-add Can you please take a look?
